### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by michal-reszka**
https://michal-reszka.github.io/layout_stars/
https://michal-reszka.github.io/layout_stars/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (mvjl000) [inline src/style.css:49]: Remember about DRY rule (don't repeat yourself)

Combine those selectors with comma `,`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  align-items: space-between;
+  height: 100%;
+  justify-content: center;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  vertical-align: middle;
```
Comment (mvjl000) [inline src/style.css:27]: vertical-align has no effect on this element since it’s not an inline or table-cell element.

tldr: you can remove this ;)

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
```
Comment (mvjl000) [inline src/style.css:10]: This value could be in a variable

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  margin-right: 4px;
```
Comment (mvjl000) [inline src/style.css:11]: Please note that the last star doesn't need a margin right

Hint: :last-child and :not 😎

**Review by mvjl000 — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 5]
```diff
@@ -23,27 +23,18 @@ body {
 .main {
   display: flex;
   justify-content: center;
-  text-align: center;
   vertical-align: middle;
```
Comment (mbruhler) [inline src/style.css:26]: vertical-align still here

**Review by mbruhler — CHANGES_REQUESTED** [review]
Hi, 
please include changes from @mvjl000

**Review by mvjl000 — CHANGES_REQUESTED** [review]
Try adding margin to last star 🤔

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  justify-content: space-between;
```
Comment (mvjl000) [inline src/style.css:12]: No need for flex. When setting background image, you just want to operate on background position etc
```suggestion
  background: url(./images/star.svg) no-repeat;
  background-position: center; // combine it with other values above
  width: 16px; // variable
  height: 16px;  //variable
  margin-right: 4px;
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
```
Comment (mvjl000) [inline src/style.css:17]: This is a default value, you don't have to specify it.
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  align-items: space-between;
```
Comment (mvjl000) [inline src/style.css:18]: This doesn't have any affect because this value for align-items doesn't exist. See [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)
```suggestion
```

**Review by mvjl000 — CHANGES_REQUESTED** [review]

**Review by mbruhler — CHANGES_REQUESTED** [review]
You didn't add last changes from @mvjl000 about moving dimensions and duplicated values into CSS Variables.

**Comment by michal-reszka** [comment]
- https://michal-reszka.github.io/layout_stars/
- https://michal-reszka.github.io/layout_stars/report/html_report/

**Review by mbruhler — APPROVED** [review]
Hi,
Good job!

Approved.

File: src/index.html [hunk 9]
```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="star-main">
```
Comment (mbruhler) [inline src/index.html:11]: Should be `stars__star` because it's an element of the `stars` block

**Review by mbruhler — CHANGES_REQUESTED** [review]
I mistakenly approved.

There is problem with BEM. Fix it and it will be accepted
